tracking.js

```diff
@@ -0,0 +1,26 @@
+const MIXPANEL_TOKEN = 'df164f13212cbb0dfdae991da60e87f2'
+
+const chalk = require('chalk')
+const mixpanel = require('mixpanel').init(MIXPANEL_TOKEN);
+
+const track = async (frontendType, contractType) => {
+    try {
+        console.log(chalk`
+NEAR collect the type of UI and contract to improve developer experience. But NEAR will never send private information.
```

Comment:
I'm not sure about the grammar here. @TiffanyGYJ can you please take a look?

Reply:
I would write something like this: 

NEAR collects basic information on the type of commands used to improve developer experiences. All data are anonymous, therefore, no personal information that could identify you will be shared.